build/yamls/antrea-aks.yml

```diff
@@ -2912,7 +2912,7 @@ metadata:
   labels:
     app: antrea
 data:
-  antrea-agent.conf: |
+  antrea-agent.conf: |+
```

Comment:
do we know where the `+` comes from? maybe it indicates some formatting issue?

Reply:
I was also puzzled on why we got that "+". couldn't find a clear reason for it. Let me try redo and check. is there any formatting option I should consider to check this?.